fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1559,6 +1559,10 @@ private void transferToNonMaster(FrontendNodeType newType) {
 
         MetricRepo.init();
 
+        if (analysisManager != null) {
+            analysisManager.getStatisticsCache().preHeat();
```

Comment:
transferToNonMaster is not only be called when fe start, but also when master changed. Could we find an appropriate place, let all fe do preHeat just when it start?